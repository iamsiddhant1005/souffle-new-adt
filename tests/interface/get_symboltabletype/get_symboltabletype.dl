.type first_name
.type last_name
.type gender

.decl people (Surname:first_name, name:last_name, gender:gender, age:number, height:number)

.input people ()
.output people ()
