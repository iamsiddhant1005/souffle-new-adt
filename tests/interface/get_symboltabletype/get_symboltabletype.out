<s:first_name,s:last_name,s:gender,i:number,i:number>

SymbolTable: {
	F	 => 9
	Hunter	 => 8
	Marry	 => 7
	Armstrong	 => 6
	Fong	 => 1
	Tom	 => 3
	Kelvin	 => 0
	M	 => 2
	Chan	 => 4
	Jack	 => 5
}

---------------
people
Surname	name	gender	age	height
===============
Kelvin	Fong	M	23	2
Tom	Chan	M	21	1
Jack	Armstrong	M	19	1
Marry	Hunter	F	20	1
===============
